src/bindings/python/src/openvino/runtime/opset9/ops.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""Factory functions for all ngraph ops."""
+from functools import partial
+from typing import Optional
+
+import numpy as np
+from openvino.runtime import Node
+from openvino.runtime.opset_utils import _get_node_factory
+from openvino.runtime.utils.decorators import nameable_op
+from openvino.runtime.utils.types import (
+    NodeInput,
+    as_nodes,
+    as_node
+)
+
+_get_node_factory_opset9 = partial(_get_node_factory, "opset9")
+
+
+# -------------------------------------------- ops ------------------------------------------------
+
+
+@nameable_op
+def eye(
+        num_rows: NodeInput,
+        output_type: str,
+        num_columns: Optional[NodeInput] = None,
+        diagonal_index: Optional[NodeInput] = None,
+        batch_shape: Optional[NodeInput] = None,
+        name: Optional[str] = None,
+) -> Node:
```

Comment:
Eye spec and shell op were changed to have only `batch_shape` as optional input.